lib/lighthouse/benefits_documents/configuration.rb

```diff
@@ -87,17 +86,13 @@ def generate_upload_body(document_data, file_body)
       File.write(fn, data.to_json)
       payload[:parameters] = Faraday::UploadIO.new(fn, 'application/json')
 
-      file_type = MimeMagic.by_path(document_data[:file_name]).subtype
-      validate_filetype!(file_type)
```

Comment:
Make sure to get rid of the `validate_filetype` method as well since we don't need it anymore

Reply:
good catch